packages/semver/package.json

```diff
@@ -6,12 +6,14 @@
   "license": "MIT",
   "main": "src/index.js",
   "builders": "./builders.json",
+  "schematics": "./collection.json",
   "repository": {
     "url": "https://github.com/jscutlery/nx-plugin-semver"
   },
   "dependencies": {
     "standard-version": "^9.0.0",
-    "@lerna/child-process": "^3.16.5"
+    "@lerna/child-process": "^3.16.5",
+    "@nrwl/angular": "^11.0.18"
```

Comment:
Do we really depend on `@nrwl/angular`?

Reply:
This package exposes the `addPackageJsonDependency` from `@schematics/angular` which is used in the schematic, it can be replaced by the direct dependency `@schematics/angular` :thinking: